tmt-app/src/Components/CareerPage.jsx

```diff
@@ -24,6 +55,54 @@ function CareerPage() {
       >
         {data.name} (Career)
       </Typography>
+      <Grid
+        container
+        spacing={4}
+        justifyContent="center"
+        sx={{ paddingX: "1rem" }}
+      >
+        {sectionInfo.map((item, index) => {
+          return (
+            <Grid item key={index} xs={12} md={6}>
+              <Card
+                sx={{
+                  boxShadow: "none",
+                  backgroundColor: `${item.color}`,
+                  height: "100%",
+                  boxSizing: "border-box",
```

Comment:
Did we need to change or explicitly declare boxSixing here?

Reply:
We need to explicitly declare boxSizing so the cards would not grow because of the padding

Reply:
MUI doesn't have this set by default?

Reply:
Actually wait don't we want this to grow because of padding?

Padding imo is extra content within the box

Reply:
Since we are using a grid, we have the cards set at the same height. We could remove it but there would be big spaces. Do you know if it is possible to make the next card to fill in the gap?
![image](https://github.com/BobaTalks/pick-a-major/assets/36683390/fd418bc1-b9a7-4cea-a9d6-21763126676f)

Reply:
Yeah it should be, what have you looked up so far?

Reply:
I know we could create staggered cards but it would be based on the custom styling instead of the card content. How would we fill the gap based on the different card content?

Reply:
You can use CSS to control the card sizes, have you given that a shot yet?

Reply:
![image](https://github.com/BobaTalks/pick-a-major/assets/36683390/174e0e6d-19e7-4d83-866b-be8734011b3c)
Yeah, I have changed the card sizes and the position but the column/row gaps is not the same with the other gaps. Notice how "related majors" card is smaller because there is less content, how would we shrink the grid height of "related majors" and move the "Any extra resources" card up to the blue box?

Reply:
Oh you want to not have a grid, I thought you wanted to make the card fill the space.

You may want to look at other component and layout strategies.

Think of a what a grid is--if it doesn't fit your use case here you may want to find another way to organize layout

Reply:
Ahh... sorry for the confusion! Okay, I will push this to a future ticket so that we could continue moving forward.